### DISCUSSION TITLE
Add support for static scalable Vector graphics SVG in graphs

### DISCUSSION THREAD

**Discussion by maurerle** [grafana/grafana#67883] [Dashboards, Panels & Visualizations]
When using graphs from grafana for scientific usage, scalable vector graphics are a must.

As already asked in https://github.com/grafana/grafana/issues/13126 and https://community.grafana.com/t/how-to-export-vectorgraph/10166 it would be very cool to be able to export a graph as SVG image in some way (through the ui or through the image rendering plugin)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). 

Most of our visualizations are rendered with canvas. Converting them to SVG for exporting would be difficult, and I don't think we will prioritizing it when other ways of exporting static images are available. If you have more ideas for how to implement this, feel free to make an [issue](https://github.com/grafana/grafana/issues) to request the feature.
